### PR TITLE
[KOGITO-8276] Fixing Dockerfile to COPY the new log package

### PR DESCRIPTION
**Pull request by davidesalerno**
**Description of the change:**

**Motivation for the change:**

**Checklist**

- [x] Add or Modify a unit test for your change
- [x] Have you verified that tall the tests are passing?

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by desmax74 — APPROVED** [review]

File: go.work.sum [hunk 1]
```diff
@@ -151,6 +151,7 @@ github.com/go-logr/logr v1.2.2/go.mod h1:jdQByPbusPIv2/zmleS9BjJVeZ6kBagPoEUsqbV
 github.com/go-logr/stdr v1.2.2 h1:hSWxHoqTgW2S2qGc0LTAI563KZ5YKYRhT3MFKZMbjag=
 github.com/go-logr/stdr v1.2.2/go.mod h1:mMo/vtBO5dYbehREoey6XUKy/eSumjCCveDpRre4VKE=
 github.com/go-logr/zapr v1.2.3/go.mod h1:eIauM6P8qSvTw5o2ez6UEAfGjQKrxQTl5EoK+Qa2oG4=
+github.com/go-logr/zapr v1.2.4/go.mod h1:FyHWQIzQORZ0QVE1BtVHv3cKtNLuXsbNLtpuhNapBOA=
```
Comment (MarianMacik) [inline go.work.sum:154]: Is this a leftover or is it needed?

**Review by MarianMacik — APPROVED** [review]
Looks good, just one question whether there is something left over or not.

**Review by radtriste — APPROVED** [review]

File: go.work.sum (hunk 1 above)
Comment (davidesalerno) [reply go.work.sum:154]: It is needed and it is generated during the `make generate-all`  by the command `go work sync`

Comment (MarianMacik) [reply go.work.sum:154]: OK. Thanks!

**Review by manstis — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**